railties/test/railties/http_request_test.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+require "isolation/abstract_unit"
+require "stringio"
+require "rack/test"
+
+module RailtiesTest
+  class HttpRequestTest < ActiveSupport::TestCase
+    include ActiveSupport::Testing::Isolation
+    include Rack::Test::Methods
+
+    def setup
+      build_app
+
+      app_file "config/routes.rb", <<-RUBY
+        Rails.application.routes.draw do
+          post 'posts', :to => 'posts#create'
```

Comment:
Let's make this style consistent with the rest of the file.

```suggestion
          post "posts", to: "posts#create"
```

Reply:
My bad 😅 my Ruby skills are a little outdated I suppose